packages/commerce/src/api/utils/errors.ts

```diff
@@ -20,3 +23,31 @@ export class CommerceNetworkError extends Error {
     this.name = 'CommerceNetworkError'
   }
 }
+
+export const getErrorMessage = (error: unknown) => {
```

Comment:
I would prefer to replace this with an error class that's very clear about what it does, like `new CommerceAPINetworkError()`

Reply:
This is used & will be used in all the try-catch of all API handlers. Now all the handlers have the error messages repeated. So I created this method because the error from catch comes as unknown, and is used only to send the text message to the response like below. This is already improved in the next PR, for the dynamic API routes. The error class will be thrown in the handlers, then in this method, we will filter what can be shown or not to the client as text messages. Now, the validation error from BigCommerce auth sends back to the res.json ‘An unexpected error occurred, so later we will have a validation error, that we send the actual message to the client, anything else we log the error and send a generic text message as unexpected error.

  ```
try {
    const body = req.body ?? {}
    return await handlers['login']({ ...ctx, body })
  } catch (error) {
    console.error(error)
    const message = getErrorMessage(error)
    res.status(500).json({ data: null, errors: [{ message }] })
  }

```

Reply:
Reverted change, this will reaper on the next PR with dynamic API routes